build.sbt

```diff
@@ -198,10 +198,23 @@ lazy val circe = crossProject(JVMPlatform, JSPlatform)
     )
   )
 
+lazy val `scala-xml` = crossProject(JVMPlatform, JSPlatform)
```

Comment:
Probably easiest to make it JVM only for now.

Reply:
That requires rearranging the tests too, since they're cross-built for js and jvm. I've tried moving the test into jvm-only sources, let's see what happens.

Reply:
Right, nothing wrong with that. Still, better not to publish for JS unless you have a cross-platform parser. scala-xml.js is kind of a lie at the moment.

Reply:
I guess we'd then need to also exclude the js version from mima? Might be better to reorganize the modules instead to do this properly